test/test_simulation_build.jl

```diff
@@ -13,7 +13,10 @@ function test_sequence_build(file_path::String)
         order = Dict(1 => "UC", 2 => "ED"),
         feedforward_chronologies = Dict(("UC" => "ED") => Synchronize(periods = 24)),
         horizons = Dict("UC" => 24, "ED" => 12),
-        intervals = Dict("UC" => Hour(24), "ED" => Hour(1)),
+        intervals = Dict(
```

Comment:
@claytonpbarrows see here that now the intervals need to define a chronology too. This allows us to know how information passes between runs of the same stage.

Reply:
@jd-lara does an interval with Consecutive() chronology always get initial conditions from the last period of the previous execution? What if your interval and horizon arent equal?

Reply:
This needs correction to make it grab the results from the end of the interval.

Reply:
solved in c752b37. Now Consecutive() chooses the last step in the interval.